workflow-examples/src/main/java/com/redhat/parodos/examples/ocponboarding/checker/JiraTicketApprovalWorkFlowCheckerTask.java

```diff
@@ -74,35 +75,39 @@ public WorkReport checkWorkFlowStatus(WorkContext workContext) {
 
 			ResponseEntity<GetJiraTicketResponseDto> result = RestUtils.restExchange(urlString + issueKey, jiraUsername,
 					jiraPassword, GetJiraTicketResponseDto.class);
-			if (result.getStatusCode().is2xxSuccessful() && result.getBody() != null) {
+			if (!result.getStatusCode().is2xxSuccessful() || result.getBody() == null) {
```

Comment:
it will return in line: https://github.com/parodos-dev/parodos/blob/mvp-demo/workflow-examples/src/main/java/com/redhat/parodos/examples/ocponboarding/checker/JiraTicketApprovalWorkFlowCheckerTask.java#L106  to also include cases with exception